docs/ai.md

```diff
@@ -56,9 +56,70 @@ For light chatbot usage we recommend that users [install Alpaca](https://flathub
 
 ## Running Ollama as a Service
 
-Ollama can also be used for people who prefer to use that tool. If you want third party tools to integrate with it, (for example an IDE) you should consider installing it [in a docker container](https://hub.docker.com/r/ollama/ollama).
+Since Alpaca doesn't expose any API, if you need other applications than Alpaca to interact with your ollama instance (for example an IDE) you should consider installing it in a [container](https://hub.docker.com/r/ollama/ollama).
 
-To do so, first configure docker to use the nvidia drivers (that come preinstalled with Bluefin) with:
+### Quadlet (recommended)
```

Comment:
```
Expected: 1; Actual: 0; Below
```
I don't know what Codacy wants here.

Reply:
Found a [link to the Codacy website for this issue](https://app.codacy.com/gh/ublue-os/bluefin-docs/pull-requests/93/issues#issue-e8a3c8c709f36995d7e33a973d2a5ff3), but there isn't much information and the suggested fix looks exactly the same